pom.xml

```diff
@@ -252,6 +274,12 @@
             <version>2.16.3</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <artifactId>commons-io</artifactId>
+            <groupId>commons-io</groupId>
+            <version>2.6</version>
+            <scope>provided</scope>
```

Comment:
I was going to leave a comment asking about bundle bloat - is it really worth it to include this when it really only saves us 4ish lines of code - but then I see that it's got a scope of provided. If we already get it for free, heck yeah we should use it! (Makes sense since it's all in the Apache family). Nice.

Reply:
yup, first thing I check is whether the necessary classes are already provided in an existing bundle.